cmd/server/assets/realmstats.html

```diff
@@ -20,7 +20,7 @@ <h1>Realm stats</h1>
       The data below shows realm statistics.
     </p>
 
-    <div class="card mb-3 shadow-sm">
+    <div class="card mb-3 shadow-sm" style="min-width:950px;">
```

Comment:
Can be a follow-up, but it'd be good to make this a class.

Reply:
Actually I think  I've fixed it. we have table-responsive which (on purpose) is for tables that should scroll as the div around them collapses. To your point, I think that is weird (especially on mobile), so I've just removed that and it seems to display properly.

We might want to remove responsive tables elsewhere too